src/Components/DialPad/dialPad.styles.scss

```diff
@@ -0,0 +1,198 @@
+.top-container{
```

Comment:
the main container should match the content size, not the screen size, this component will be used as a child of other components